packages/trader/src/Modules/CFD/Containers/cfd-financial-stp-real-account-signup.tsx

```diff
@@ -7,20 +6,89 @@ import { connect } from 'Stores/connect';
 import CFDPOA from '../Components/cfd-poa.jsx';
 import CFDPersonalDetailsForm from '../Components/cfd-personal-details-form';
 import CFDPOI from '../Components/cfd-poi';
+import { LandingCompany, ResidenceList, GetSettings, StatesList } from '@deriv/api-types';
+import RootStore from 'Stores/index';
 
-const index_lookup = {
+type TAuthenticationStatus = { document_status: string; identity_status: string };
+
+type TStoreProofOfAddressArgs = {
+    file_uploader_ref: HTMLDivElement | null;
+    values: { [key: string]: string };
+};
+
+type TRemoveNotificationMessage = {
+    key: string;
+    should_show_again: boolean;
+};
+
+type TIndexLookupObject = {
+    CFDPersonalDetailsForm: number;
+    CFDPOI: number;
+    CFDPOA: number;
+    CFDPendingVerification: number;
+};
+
+type TGetSettings = GetSettings & {
+    upload_file?: string;
+    poi_state?: string;
+};
+
+type TCFDFinancialStpRealAccountSignupProps = {
+    addNotificationByKey: (key: string) => void;
+    authentication_status: () => TAuthenticationStatus;
+    get_settings: TGetSettings;
+    client_email: string;
+    is_fully_authenticated: boolean;
+    landing_company: LandingCompany;
+    openPendingDialog: () => void;
+    refreshNotifications: () => void;
+    removeNotificationMessage: () => void;
+    removeNotificationByKey: (args: TRemoveNotificationMessage) => void;
+    residence_list: ResidenceList;
+    states_list: StatesList;
+    storeProofOfAddress: TStoreProofOfAddressArgs;
+    toggleModal: () => void;
+};
+
+type TSetSubmiting = (isSubmitting: boolean) => void;
+
+type TNextStep = (submitting: TSetSubmiting) => void;
+
+type TItemsState = {
+    header: { [key: string]: string };
+    body: typeof CFDPersonalDetailsForm | typeof CFDPOI | typeof CFDPOA;
+    form_value: { [key: string]: string | undefined };
+    props: Array<string>;
+};
+
+type TItemsProps =
+    | 'residence_list'
+    | 'is_fully_authenticated'
+    | 'landing_company'
+    | 'addNotificationByKey'
+    | 'authentication_status'
+    | 'refreshNotifications'
+    | 'removeNotificationMessage'
+    | 'removeNotificationByKey'
+    | 'states_list'
+    | 'get_settings'
+    | 'storeProofOfAddress'
+    | 'refreshNotifications';
```

Comment:
@mahdiyeh-fs do we need this duplicate of line 70?

Reply:
my bad, will fix